src/selectors/user-list/index.js

```diff
@@ -0,0 +1,11 @@
+export const sortUser = (state) => state.user.tempUsers = state.user.tempUsers.sort(compareObjects);
```

Comment:
also write `selectUserFromState` selector that will just return `state.user`. This is required because if in future, user is moved to some other state tree, then instead of changing the dot access everywhere, we'd only be changing the base selector of `selectUserFromState`

Reply:
use that selectors in all of the selectors and do the processing based on the result of base selector